readme.md

```diff
@@ -62,28 +73,49 @@ Alternatively, you can set `MICROPUB_LOCAL_AUTH` to 1 to use WordPress's interna
 Finally, for ease of development, if the WordPress site is running on `localhost`, it logs a warning if the access token is missing or invalid and still allows the request.
 
 
-### Troubleshooting
+
+## Installation 
+
+Install from the WordPress plugin directory or put `micropub.php` in your plugin directory. No setup needed.
+
+
+
+## Configuration Options 
+
+These configuration options can be enabled by adding them to your wp-config.php
+
+* `define('MICROPUB_LOCAL_AUTH', '1')` - Bypasses Micropub authentication in favor of WordPress authentication
+* `define('MICROPUB_AUTHENTICATION_ENDPOINT', 'https://indieauth.com/auth')` - Define a custom authentication endpoint
+* `define('MICROPUB_TOKEN_ENDPOINT', 'https://tokens.indieauth.com/token')` - Define a custom token endpoint
+* `define('MICROPUB_DRAFT_MODE', '1')` - Set all Micropub posts to draft mode
+
+
+## Frequently Asked Questions 
 
 If your Micropub client includes an `Authorization` HTTP request header but you still get an HTTP 401 response with body `missing access token`, your server may be stripping the `Authorization` header. If you're on Apache, [try adding this line to your `.htaccess` file](https://github.com/snarfed/wordpress-micropub/issues/56#issuecomment-299202820):
 
-```
-SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
-```
+    SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
 
 If that doesn't work, [try this line](https://github.com/georgestephanis/application-passwords/wiki/Basic-Authorization-Header----Missing):
 
-```
-RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
-```
+    RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+
+If that doesn't work either, you may need to ask your hosting provider to whitelist the `Authorization` header for your account. If they refuse, you can [pass it through Apache with an alternate name](https://github.com/snarfed/wordpress-micropub/issues/56#issuecomment-299569822), but [you'll need to edit this plugin's code to read from that alternate name](https://github.com/snarfed/wordpress-micropub/issues/56#issuecomment-299569822).
+
 
-If that doesn't work, you may need to ask your hosting provider to whitelist the `Authorization` header for your account. If they refuse, you can [pass it through Apache with an alternate name](https://github.com/snarfed/wordpress-micropub/issues/56#issuecomment-299569822), but [you'll need to edit this plugin's code to read from that alternate name](https://github.com/snarfed/wordpress-micropub/issues/56#issuecomment-299569822).
+## Upgrade Notice 
 
+None yet.
 
-### License
```

Comment:
this section needs to be preserved.

Reply:
This means, again, it wasn't in both files. I'm going to do a piece by piece comparison and edit the readme and rerun the converter to include all.